spec/scryfall/cards_spec.rb

```diff
@@ -28,20 +28,33 @@
         end
       end
     end
+  end
 
-    context 'random' do
-      it 'returns a JSON response containing the data for a random card' do
-        VCR.use_cassette('successful random search') do
-          expect(Scryfall::Cards.random(query: nil)['object']).to eq('card')
-        end
+  context 'random' do
+    it 'returns a JSON response containing the data for a random card' do
+      VCR.use_cassette('successful random search') do
+        expect(Scryfall::Cards.random(query: nil)['object']).to eq('card')
       end
     end
 
-    context 'random' do
-      it 'returns a JSON response containing the data for a filtered random card search' do
-        VCR.use_cassette('successful filtered random search') do
-          expect(Scryfall::Cards.random(query: 'Jace')['name']).to start_with 'Jace'
-        end
+    it 'returns a JSON response containing the data for a filtered random card search' do
+      VCR.use_cassette('successful filtered random search') do
+        expect(Scryfall::Cards.random(query: 'Jace')['name']).to start_with 'Jace'
+      end
+    end
+  end
```

Comment:
I moved both tests for `self.random` into the same context for organizational purposes.